getnf

```diff
@@ -26,6 +27,12 @@ while getopts :hk option; do
     esac
 done
 
+# For Macs, need to set a few different things
+if [[ "$os" == 'Darwin' ]]; then
+    distDir="$HOME/Library/Fonts"
+    [ -e "$downDir" ] && echo "Fonts Directory exists, good" || mkdir -p "$downDir"
```

Comment:
this is redundant, as curl can create a directory with the `--create-dirs` flag

Reply:
I’ll try again but it was not working for me on mac

Reply:
Updated